tests/test_inference_providers.py

```diff
@@ -324,6 +324,53 @@ def test_prepare_request_conversational(self):
             "messages": [{"role": "user", "content": "dummy text input"}],
         }
 
+    @pytest.mark.parametrize(
+        "mapped_model,parameters,expected_model",
+        [
+            (
+                "username/repo_name",
+                {},
+                "username/repo_name",
+            ),
+            # URL endpoint with model in parameters - use model from parameters
+            (
+                "https://localhost:8000/v1/chat/completions",
```

Comment:
```suggestion
                "http://localhost:8000/v1/chat/completions",
```

Let's test on http:// otherwise it's the same test as `# HTTPS endpoint with model in parameters` below